### ISSUE TITLE
Issue/16067: PFF Updates: ACS-Economic: top and bottom coded values should be grey 

### ISSUE THREAD

**Issue by horatiorosa** [NYCPlanning/labs-factfinder#1178]
#### Description
Top and bottom coded values should be grey for selected and comparison area and both time points in change over time

[Example here]( https://factfinder-staging.planninglabs.nyc/explorer/selection/4d02a8ed2116f88de221484728bd0be6800419bd?acsTopics=econ-incomeAndBenefits&censusTopics=populationSexAgeDensity%2CmutuallyExclusiveRaceHispanicOrigin%2CdetailedRaceAndEthnicity%2CrelationHeadHousehold%2ChouseholdType%2ChousingOccupancy%2ChousingTenure%2ChouseholdSize&compareTo=0&showCharts=true&showReliability=false&source=acs-change
)

#### Note
Make a branch from staging and merge all PR back into staging

**Comment by TylerMatteo** [comment]
@horatiorosa to verify if this is a new issue before completing.

**Comment by fvankrieken** [comment]
Though per @TylerMatteo's comment above, this is not new, and is currently on production as well

See [here](https://popfactfinder.planning.nyc.gov/explorer/selection/9d6c96be41b393f5bf80aecbc0d71987a70c3d86?acsTopics=%2Cecon-incomeAndBenefits&source=acs-current)

**Comment by horatiorosa** [comment]
@TangoYankee, do you recall this issue being brought to your attention during the last round of PFF updates?

**Comment by TylerMatteo** [comment]
> So in the case that a row is coded but `isReliable` is true, it's never getting properly set as false
> 
> In [recalculateSpecialVariables](https://github.com/NYCPlanning/labs-factfinder-api/blob/b823ba0547cf372637eab2ee01881e17e8407990/utils/data-processor.js#L65), we call (among other things) in this order
> 
>     * [recalculateCorrelationCoefficient](https://github.com/NYCPlanning/labs-factfinder-api/blob/b823ba0547cf372637eab2ee01881e17e8407990/utils/data-processor.js#L76)
> 
>     * [recalculateIsReliable](https://github.com/NYCPlanning/labs-factfinder-api/blob/b823ba0547cf372637eab2ee01881e17e8407990/utils/data-processor.js#L77)
> 
> 
> For a coded row, recalculateCorrelationCoefficient [sets `row.correlationcoefficient` to `null` ](https://github.com/NYCPlanning/labs-factfinder-api/blob/b823ba0547cf372637eab2ee01881e17e8407990/utils/data-processor.js#L166) Then, in recalculateIsReliable, if `row.correlationcoefficient` is `null`, [the function returns immediately, not setting `isReliable` to False if row `wasCoded`](https://github.com/NYCPlanning/labs-factfinder-api/blob/b823ba0547cf372637eab2ee01881e17e8407990/utils/data-processor.js#L177)
> 
> I do think that potentially these rows should not have `isReliable` as true in the first place, but regardless, this seems to be a bug

Yeah so it looks like the immediate return was added by [PR 204](https://github.com/NYCPlanning/labs-factfinder-api/pull/204) on the backend [which fixes a bug unrelated to coded values](https://dev.azure.com/NYCPlanning/ITD/_workitems/edit/10016). Maybe the best way to preserve that change while fixing this apparent regression would be to change line 177 from `if (!row.correlationCoefficient) return;` to `if (!row.correlationCoefficient && !wasCoded) return;`. That way, coded rows would proceed on to the line that sets isReliable to false.
